src/sudoku.js

```diff
@@ -66,6 +66,29 @@ export const play = ({
   });
 };
 
+export const erase = ({ puzzle, colChar, rowChar, author }) => {
+  const [col, row] = getCoordinates(colChar, rowChar);
+  return puzzle.map((_, indexRow) => {
+    return _.map((cell, indexCol) => {
```

Comment:
No need to map, we can do like in blame (it should be the same in play actually).